google-cloud-bigquery/lib/google/cloud/bigquery/copy_job.rb

```diff
@@ -176,6 +178,33 @@ def self.from_options source, target, options = {}
             updater
           end
 
+          ##
+          # Sets the geographic location where the job should run. Required
+          # except for US and EU.
+          #
+          # @param [String] value A geographic location code, such as "US", "EU"
```

Comment:
nit: I would remove "code" here. We typically refer to it as "geographic location".

Reply:
Thanks, fixed everywhere.